apps/dot-voting/app/Decisions.js

```diff
@@ -166,7 +166,7 @@ class Decisions extends React.Component {
                 icon={<EmptyIcon />}
                 title="You do not have any dot votes."
                 text="Use the Allocations app to get started."
-                actionButton={() => <div />}
+                onActivate={() => <div />}
```

Comment:
Was this just a typo you noticed while you were here?

Reply:
No, this started appearing in warnings - actionButton being obsolete. It should be a separate PR, I know, but...

Reply:
Oh, that seems related to me! We updated the library and fixed deprecation warnings. 👍